packages/flutter_test/lib/src/matchers.dart

```diff
@@ -1973,10 +2237,38 @@ class _MatchesSemanticsData extends Matcher {
       description.add(' with tooltip: $tooltip');
     }
     if (actions != null) {
```

Comment:
`actions` as specified in line 2190 is always non-null now. Do you still need this?

Reply:
Good catch, missed that one. Thanks!